msgraph-developer-proxy/responses.json

```diff
@@ -1,7 +1,7 @@
 {
   "responses": [
     {
-      "url": "/v1.0/me",
+      "url": "https://graph.microsoft.com/v1.0/me",
```

Comment:
Should we have an example url in the responses that shows using a wildcard in the hostname? 

e.g. "url": "https://graph.microsoft.*/v1.0/me",

Reply:
We've got `https://graph.microsoft.com/v1.0/users/*/photo/$value` and `https://graph.microsoft.com/v1.0/users/*`. Do you think it would be valuable to add one with a wildcard in the hostname as well? Btw, we're not shipping this file in the release, so it's only in this repo for testing. Otherwise, it's something that we'd explain more in detail in our docs.

Reply:
You're right, I'd forgotten that we were not shipping this file, this is all good IMO.